### DISCUSSION TITLE
Custom Domain Resolution

### DISCUSSION THREAD

**Discussion by dezren39** [acorn-io/runtime#570] [General]
Hello! If I used my own domain, is there DNS resolution? Or is there another tool that can be used with acorn which would let me recreate on-acorn? How does that all work ?

**Comment by cjellick** [comment]
Unfortunately, you don't get DNS resolution if you use your own domain.

For our own production services using acorn (which can't us on-acorn.io), we are using custom domains per-service and setting up the DNS semi-manually to point at the public interface of our clusters.

on-acorn.io is powered by this project: https://github.com/acorn-io/acorn-dns. I just wrote a blog about how we develop and deploy it here: https://acorn.io/how-were-using-acorn-here-at-acorn-labs/. So, if you really wanted to, you could get it working for your own domain. You would have to run your own instance of acorn-dns and configure your install of acorn to use it. It's all possible, but would be pretty advanced :-). I'll be the first to admit the project is not well documented.
